terragrunt/aws/cloudfront/certificates.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_acm_certificate" "simplify_privacy_statement_certificate" {
+  provider = aws.ca-central-1
+
+  domain_name               = var.domain
+  subject_alternative_names = ["*.${var.domain}"]
+  validation_method         = "DNS"
+
+  tags = {
+    CostCentre = var.billing_code
+    Terraform  = true
+  }
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_route53_record" "simplify_privacy_statement_dns_validation" {
```

Comment:
This resource will cause the Route53 hosted zone `TXT` records to be created, which lets AWS know we control the domain and they're safe to issue a certificate.